docs/reference/modules.rst

```diff
@@ -10,3 +10,4 @@ Modules
 
    constants
    spatial
+   logging
```

Comment:
Is there anything user-facing in here? I see key functions are imported directly.

Reply:
All the public functions can be useful for setting up custom loggers, see the reference page. I don't see a downside of showing it in the docs, especially because it is at the bottom and out of the way.